Makefile

```diff
@@ -65,6 +65,9 @@ createuser: compose_build
 load_locales_countries: compose_build
 	docker-compose run app python manage.py load-locales-countries
 
+load_dummy_experiments: compose_build
+	docker-compose run app python manage.py load-dummy-experiments
+	
```

Comment:
ooh should the optional params be available here too?

Reply:
@tiftran Yeah I've tried that, I couldn't figure out a nice way to do it, I wouldn't worry about it, if you wanna use the parameters you can do it from within the containers.

Also while you're at it, add a makefile command here like

    refresh: kill migrate load_locales_countries load_dummy_experiments

I feel like I do that all the time so having a shortcut would be hand ;)